src/main/java/dev/cortex/cortexbot/commands/LeaderboardCommand.java

```diff
@@ -30,27 +29,30 @@ protected void execute(SlashCommandEvent event) {
                 .stream()
                 .sorted(Comparator.comparing(CortexMember::getPoints).reversed())
                 .limit(15)
-                .collect(Collectors.toList());
+                .toList();
 
-        MessageCreateBuilder message = new MessageCreateBuilder();
+        EmbedBuilder builder = new EmbedBuilder();
 
-        message.addContent("---------------------------------------------------------------------------------------------").addContent("\n");
-        message.addContent("\uD83D\uDE80 **Top 15 Leaderboard Rankings** \uD83D\uDE80").addContent("\n\n");
+        builder
+                .setColor(Color.CYAN)
+                .setTitle("**Top 15 Leaderboard Rankings**")
+                .addBlankField(false);
 
         for (int i = 0; i < top.size(); i++){
-            message.addContent("(" + (i + 1) + ") - ").addContent("<@" + top.get(i).getUserID() + "> *-* " + top.get(i).getPoints() + " pts");
-            if (i == 0) message.addContent(":first_place:");
-            if (i == 1) message.addContent(":second_place:");
-            if (i == 2) message.addContent(":third_place:");
-            message.addContent("\n");
+
+            StringBuilder nameString = new StringBuilder("(" + (i + 1) + ")  <@" + top.get(i).getUserID() + ">");
+            if (i == 0) nameString.append(":first_place:");
+            if (i == 1) nameString.append(":second_place:");
+            if (i == 2) nameString.append(":third_place:");
+
+            builder.addField(nameString.toString(), top.get(i) + "pts", false);
```

Comment:
You are doing top.get(i) here on an object rather than primitive value

Reply:
![image](https://github.com/Cortex-Development/cortex-bot/assets/77004702/c6e86aad-518b-46f6-8adf-07ad74cbb1d3)